site-src/v1alpha2/guides/tls.md

```diff
@@ -130,28 +130,6 @@ there are two HTTPRoute resources which specify certificates for
 {% include 'tls-cert-in-route.yaml' %}
 ```
 
-## Upstream TLS
-
-Upstream TLS configuration applies to the connection between the Gateway
-and Service.
-
-There is only one way to configure upstream TLS: using the `BackendPolicy`
-resource.
-
-Please note that the TLS configuration is related to the Service or backend
-resource and not related to a specific route resource.
```

Comment:
Can we keep some of this text and inform the user to use ReferencePolicy for these purposes? it could help with some education and getting our users and implementers be more aware about the new feature.

Reply:
I think that the Policy Attachment GEP is likely a better fit here, though it will be easier to reference that when #736 merges.

Reply:
Added a hold on #736 to deal with this there.